src/main.cpp

```diff
@@ -24,46 +14,36 @@ Dispatcher *dispatcher;
 unsigned long lastPublish = 0;
 
 void publishMessage() {
-    long start, json_build_time;
-    if (DEBUG_CPU_TIME) {
-        start = micros();
-    }
 
-    if (DEBUG_CPU_TIME) {
-        json_build_time = micros() - start;
-    }
+    // dataQ.add("property", "value");
 
     DEBUG_SERIAL_LN("------------------------");
     DEBUG_SERIAL_LN("Time: " + Time.timeStr());
     if(PUBLISH_ENABLED){
-        DEBUG_SERIAL_LN("URBAN - Publish ENABLED - Message: ");
+        DEBUG_SERIAL_LN(String(VEHICLE_NAME) + " - Publish ENABLED - Message: ");
         // Publish to Particle Cloud
-        DEBUG_SERIAL_LN(dataQ.publish("Proto", PRIVATE, WITH_ACK));
+        DEBUG_SERIAL_LN(dataQ.publish("BQIngestion", PRIVATE, WITH_ACK));
     }else{
-        DEBUG_SERIAL_LN("URBAN - Publish DISABLED - Message: ");
+        DEBUG_SERIAL_LN(String(VEHICLE_NAME) + " - Publish DISABLED - Message: ");
         DEBUG_SERIAL_LN(dataQ.resetData());
     }
-
-    CurrentVehicle::publishMessage();
+    
+    if(DEBUG_SENSOR_ENABLE){
+        CurrentVehicle::debugSensorData();
+    }
 
     if(DEBUG_MEM){
         DEBUG_SERIAL_LN("\nFREE RAM: " + String(System.freeMemory()) + "B / 128000B");
     }
 
-    // Output CPU time in microseconds spent on each task
-    if (DEBUG_CPU_TIME) {
-        DEBUG_SERIAL("\nCPU Time:");
-        DEBUG_SERIAL("Build JSON Message: " + String(json_build_time) + "us");
-        for (unsigned i = 0; i < sensor_count; i++) {
-            DEBUG_SERIAL(sensors[i]->getHumanName() + " polling: " + String(sensors[i]->getLongestHandleTime()) + "us");
-        }
-        DEBUG_SERIAL_LN();
-    }
 }
 /**
  * SETUP
  * */
 void setup() {
+    static_assert(CURRENT_VEHICLE == PROTO || CURRENT_VEHICLE == URBAN || CURRENT_VEHICLE == FC,
```

Comment:
Check to make sure a correct vehicle has been chosen. If not, project should not compile.

Reply:
I don't think we need this static assert since we have the define guards in the implementations; if none of the implementations are selected, the linker won't be able to find proper definitions and will fail to compile.

Reply:
I've removed it